clients/client-python/gravitino/api/decimal.py

```diff
@@ -0,0 +1,59 @@
+from decimal import Decimal as PyDecimal, ROUND_HALF_UP
+from typing import Union
+from .types import Types
+
+
+class Decimal:
```

Comment:
Here is the reason why we need to wrap Java `BigDecimal`:
https://github.com/apache/gravitino/blob/6d05ec4b8bf278c2dd7a36af35589f07b220fcab/api/src/main/java/org/apache/gravitino/rel/types/Decimal.java#L27-L34

Does the `PyDecimal` have the same issue?

Reply:
Thank you so much to point out the reason why we need to wrap Java BigDecimal. Then I just double checked for Python PyDecimal, it tends to handle this more intuitively for currency by counting all visible digits. So for example:
```
from decimal import Decimal

value = Decimal("0.01") ## it is 2
print("Precision:", value.adjusted() + 1)  ## it is 2 
print("Scale:", -value.as_tuple().exponent)
```
So PyDecimal didn't have the similar issue like Java BigDecimal. 
I will remove the new change for decimal.py.

Reply:
Removed. Done.